### ISSUE TITLE
Many CSP errors while loading the widget iframe on firefox 55

### ISSUE THREAD

**Issue by mrmakeit** [codesy/widgets#114]
After installing the widget (had to manually install it due to #113), I receive multiple CSP errors when visiting github issues, and was not seeing the iframe until after I posted a new issue.

Sample Errors: 

```Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://www.codesy.io https://cdn.jsdelivr.net https://*.stripe.com https://www.google-analytics.com”). Source: call to eval() or related function blocked by CSP.```

```Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src https://www.codesy.io https://assets-cdn.github.com”). Source: call to eval() or related function blocked by CSP.```

**Comment by jdungan** [comment]
@mrmakeit Did you visit codesy (local or .io) after installing the widget manually?

**Comment by mrmakeit** [comment]
Yes.

**Comment by groovecoder** [comment]
According to report-uri.com, the latest errors are coming from `data:` uri's. I suspect that jQuery and/or Foundation are responsible.
